doc/classes/Array.xml

```diff
@@ -446,7 +447,8 @@
 			<argument index="0" name="value" type="Variant">
 			</argument>
 			<description>
-				Adds an element at the beginning of the array.
+				Adds an element at the beginning of the array. See also [method push_back].
+				[b]Note:[/b] On large arrays, this method is much slower than [method pop_back] as it will reindex all the array's elements every time it's called. The larger the array, the slower [method push_front] will be.
```

Comment:
```suggestion
				[b]Note:[/b] On large arrays, this method is much slower than [method push_back] as it will reindex all the array's elements every time it's called. The larger the array, the slower [method push_front] will be.
```